numpy/lib/tests/test_nanfunctions.py

```diff
@@ -22,6 +22,18 @@
          np.array([0.1042, -0.5954]),
          np.array([0.1610, 0.1859, 0.3146])]
 
+# Rows of _ndat with nans converted to ones
+_rdat_ones = [np.array([0.6244, 1.0, 0.2692, 0.0116, 1.0, 0.1170]),
```

Comment:
I think this should probably be all one big array -- it's only not one array for `_rdat` because `_rdat` is ragged